components/gcp-click-to-deploy/src/Gapi.ts

```diff
@@ -0,0 +1,105 @@
+export default class Gapi {
+
+  public static deploymentmanager = class {
+
+    public static async insert(project: string, resource: {}) {
+      await this._load();
+      return this._deploymentManager.insert({ project, resource } as any).then(r => r.body);
+    }
+
+    public static async list() {
+      await this._load();
+      return this._deploymentManager.list({ project: 'yelsayed-project1' }).then(r => r.body);
```

Comment:
Did you mean to leave your project here?

Reply:
Just leaving my mark on the project! :)
Thanks for the catch, removed.